root/user/UserProfile.js

```diff
@@ -875,6 +875,20 @@ component UserProfile(
   if (adminViewing && isUntrusted(user)) {
     restrictions.push(l_admin('Untrusted'));
   }
+  const searchEditNotesURL =
+    '/search/edits' +
+    '?conditions.0.field.0=editor' +
+    `&conditions.0.args.0=${user.id}` +
+    `&conditions.0.name=${encodedName}` +
+    '&conditions.0.operator=%3D' +
+    '&conditions.1.field=edit_note_author' +
+    `&conditions.1.args.0=${$c?.user?.id ?? ''}` +
+    `&conditions.1.name=${encodeURIComponent($c?.user?.name ?? '')}` +
+    '&conditions.1.operator=%3D' +
+    '&combinator=and' +
+    '&order=desc' +
+    '&negation=0' +
+    '&auto_edit_filter=';
```

Comment:
It should probably be just set to `''` when unused, and when viewing our own profile it could be used to search edit notes left by other editors on our own edits.

Reply:
Sure, please let me know if what I have now is what you were thinking.